nemo/export/tensorrt_llm.py

```diff
@@ -202,6 +204,8 @@ def export(
             multiple_profiles: (bool): enables multiple profiles feature of TRT-LLM. Default = False
             gpt_attention_plugin (str): enable the gpt attention plugin. Default = "auto"
             gemm_plugin (str): enable the gpt plugin. Default = "auto"
+            fp8_quantized (bool): enables exporting to FP8 TRT-LLM checkpoints
+            fp8_kvcache (bool): enables FP8 KV-cache quantization
```

Comment:
Following up on Onur's comment: What happens if these are not set correctly?
Perhaps if would be more user friendly to make these optional, so if these are not set then we try to detect the data type format of the checkpoint. The function signature of this function is already pretty huge

Reply:
For FP8 checkpoints:
* `fp8_quantized=False`, creates a non-quantised checkpoint
* `fp8_kvcache=False`, disables KV-cache quantisation

For non-FP8 checkpoints:
* `fp8_quantized=True`, TRT-LLM fails to build with `Required, but not provided` error regarding scaling factors
* `fp8_kvcache=True`, exports the engine with quantised KV-cache

As suggested, I've made them optional with a new commit (and by default auto-detect)